BottomsUp/app/src/test/java/edu/uc/brown5a7/BottomsUp/DrinkDataIntegrationTest.kt

```diff
@@ -17,19 +22,26 @@ class DrinkDataIntegrationTest {
 
     var drinkService = mockk<DrinkService>()
 
+    @Before
+    fun setUp()
+    {
+        mvm = MainViewModel()
+    }
+
     @Test
     fun searchForCoke_returnsCoke(){
-        givenAFeedOfDrinkDataAreAvailable()
-        whenSearchForCoke()
+        whenSearchingFor("Coke")
         thenResultContainsCoke()
     }
 
-    private fun givenAFeedOfDrinkDataAreAvailable() {
-        mvm = MainViewModel()
+    private fun whenSearchingFor(searchTerm: String)
+    {
+        mvm.fetchDrinks(searchTerm)
     }
 
-    private fun whenSearchForCoke() {
-        mvm.fetchDrinks("coke")
+    private fun whenSearchingForCatagory(searchCatagory: String)
+    {
+        mvm.fetchCategory(searchCatagory)
     }
```

Comment:
Added in functions that use the fetchDrinks and fetchCategory functions that pass in a parameter. Removes the need for multiple when functions to search for a specific term

Reply:
"Category" is the proper spelling.